### DISCUSSION TITLE
When I edit my settings file, it says `You have an error in your yaml syntax on line 70`

### DISCUSSION THREAD

**Discussion by patstha** [upptime/upptime#921] [Q&A] [answered]
It says on https://github.com/patstha/upptime/actions/runs/7626044706

Annotations
1 error
Invalid workflow file: .github/workflows/response-time.yml#L70
You have an error in your yaml syntax on line 70

![image](https://github.com/upptime/upptime/assets/56666486/8e76b655-5017-4141-addf-f36ab5c2574c)

Can someone please take a look and help me with this issue? 
Maybe I have a syntax error in my yaml file?

**Comment by patstha** [comment]
I will create an organization and try there

**Comment by andrewgioia** [comment]
I'm seeing a similar error in `setup.yml` on a brand new deployment this morning:

<img width="589" alt="Screenshot 2024-01-24 at 9 40 07 AM" src="https://github.com/upptime/upptime/assets/221473/91503850-8626-4642-98b3-9771c74c00a4">

**Reply by patstha** [reply, marked as answer]
yes, I think this is an issue now and no longer a support request :) 

for anyone who stumbles upon this question, please see https://github.com/upptime/upptime/issues/922
